media-plugins/vdr-newsticker/vdr-newsticker-0.0.4-r1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit user-info vdr-plugin-2
+
+DESCRIPTION="VDR plugin: Show rdf Newsticker on TV"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI="http://vdr.websitec.de/download/${PN}/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+
+BDEPEND="acct-user/vdr"
+DEPEND="${BDEPEND}
+	>=media-video/vdr-1.2.6
+	net-misc/wget"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/${P}-gcc4.diff" )
+QA_FLAGS_IGNORED="
+	usr/lib/vdr/plugins/libvdr-.*
```

Comment:
I don't think, this is correct, as this is a source based package, so why ignore *flags?

Reply:
All packages media-plugins/vdr-* are build with the CFLAGS taken from the BDEPENDed media-video/vdr. This may result into a QA message that this package is ignoring the current CFLAGS.
Copying the CFLAGS from main vdr package is necessary to asssure that the plugins (libraries) match exactly.